src/ui/components/Label.qml

```diff
@@ -23,6 +23,32 @@ Grid {
         leftPadding: 0;
         verticalAlignment: Text.AlignVCenter;
         height: root.position === Label.Top? undefined : inner.height;
+        MouseArea {
+            id: ma;
+            anchors.fill: t;
+            hoverEnabled: true;
+            acceptedButtons: Qt.LeftButton | Qt.RightButton;
+            propagateComposedEvents: true;
+            preventStealing: true;

```

Comment:
Are all these needed? Also is double right click a thing? I would just leave `acceptedButtons: Qt.LeftButton` instead of these 4 lines.

Reply:
fixed now